packages/cli/api-importers/openapi/openapi-v2-importer/src/3.1/paths/ParameterConverter.ts

```diff
@@ -67,6 +68,15 @@ export class ParameterConverter extends AbstractConverter<OpenAPIConverterContex
         context: OpenAPIConverterContext3_1;
         errorCollector: ErrorCollector;
     }): ParameterConverter.Output | undefined {
+        const shouldIgnore = new FernIgnoreExtension({
+            breadcrumbs: this.breadcrumbs,
+            operation: this.parameter
+        }).convert({ context, errorCollector })?.ignore;
+
+        if (shouldIgnore) {
+            return undefined;
+        }
```

Comment:
@eyw520 id love to define this logic in one place instead of having to implement everywhere. 


One thought i have is that the `x-fern-ignore` extension runs as the first step in the `OpenAPIConverter` and takes the entire document as input and iterates over every key/value. At any key where there is `x-fern-ignore`, we just delete that key,value from the document

Reply:
This would improve the UX for users as well b/c we can just tell them to add `x-fern-ignore` anywhere and we'll just ignore that entire key